ocaml/xapi-consts/api_errors.ml

```diff
@@ -316,8 +316,10 @@ let pgpu_insufficient_capacity_for_vgpu = "PGPU_INSUFFICIENT_CAPACITY_FOR_VGPU"
 let vgpu_type_not_enabled = "VGPU_TYPE_NOT_ENABLED"
 let vgpu_type_not_supported = "VGPU_TYPE_NOT_SUPPORTED"
 let vgpu_type_not_compatible_with_running_type = "VGPU_TYPE_NOT_COMPATIBLE_WITH_RUNNING_TYPE"
+let vgpu_type_not_compatible = "VGPU_TYPE_NOT_COMPATIBLE"
 let vgpu_destination_incompatible = "VGPU_DESTINATION_INCOMPATIBLE"
 let nvidia_tools_error = "NVIDIA_TOOLS_ERROR"
+let vgpu_vm_ran_out_of_pci_slot = "VGPU_VM_RUN_OUT_OF_PCI_SLOT"
```

Comment:
I'm changing this to `vm_pci_bus_full` so that we can use the same error for other PCI devices as well.